config/environments/staging.rb

```diff
@@ -113,4 +113,8 @@
   config.disable_admin_notifications = true
 
   config.bard_host_url = 'https://terra-bard-dev.appspot.com'
+
+  # Host header injection protection
+  # from https://guides.rubyonrails.org/configuring.html#configuring-middleware
+  config.hosts << ENV['PROD_HOSTNAME']
```

Comment:
outside the scope of this change -- but we should probably rename this environment variable to something like `APP_SERVER_HOSTNAME` so that statements like the above don't cause double-takes -- "Why is he setting the staging host to prod?"

Reply:
Agreed, I also want to rename `PROD_DB_PASSWORD` in `scp_config.json`.